dart/lib/src/enricher/web_enricher_event_processor.dart

```diff
@@ -55,7 +54,7 @@ class WebEnricherEventProcessor extends EventProcessor {
     );
   }
 
-  Future<SentryDevice> _getDevice(SentryDevice? device) async {
```

Comment:
@ueman do you remember why this was a Future? apparently, there's no reason.

Reply:
Nope, no idea. I probably tried to get more information out of the navigator. There are a few things on the navigator which need to be awaited.